README.md

```diff
@@ -44,34 +41,26 @@ sudo chmod +x some-qt-build.run
 ./some-qt-build.run
 ```
 
-**2** Setup QMAKE environment variable to path to qmake
-```export QMAKE=${PATH_TO_QMAKE} ```
-*You can get path to qmake this way:*
-```
-$ sudo find / -name qmake
-/usr/bin/qmake
-/opt/Qt5.3.1/5.3/gcc/bin/qmake
-```
-/usr/bin/qmake in most cases does not work as it is just soft link to qtchooser
-
-**2**  ```$ make```
-
- 1) If during executing "make" command you have the following problem:
-   Project ERROR: Unknown module(s) in QT: websockets
-
-   Solution:
-   You have to change location of qmake in Makefile in atf root directory
-   Find location of qmake executable on your local PC:
-   (It should look like: .../Qt/5.3/gcc_64/bin/qmake)
+**3** Build ATF
+- Create build directory and get into it
+- Run `cmake <path_to_sources>`
+- Run `make`
 
-   and put it into Makefile to the line:
-   QMAKE=<your path to qmake>
-   Sometimes you will need reinstall QT Creator to get correct qmake executable
-   Also you can open QT Creator. Then go to: Tool->Options->Build & Run. Find Qt Versions Tab.
-   Here you can find qmake location.
+## Configuration of ATF
+ATF configuration is setting up in `modules/configuration` folder.
+- `base_config.lua` : base configuration parameters (reporting, paths to SDL e.t.c)
+- `connection_config.lua` : configuration parameters related to all connections (mobile, hmi, remote)
+- `security_config.lua` : configuration parameters related to security layer of connection
+- `app_config.lua` : predefined applications parameters
+Each folder in this folder represents values of `--config` option for ATF run: `local`, `remote_linux`, `remote_qnx`
+They can override one or more described configuration files.
 
 ## Run:
-``` ./start.sh [options] [script file name] ```
+```
+./start.sh [--config=<config_name>] [options] [script file name]
+```
+
+where <config_name> is one of the following values: `local`, `remote_linux`, `remote_qnx`
 
```

Comment:
Should also add additional instructions for running the `RemoteTestingAdapterServer` on the host machine before starting the test with the remote config

Reply:
@ShobhitAd Please find update in ef4c36d